Resources/Prototypes/Recipes/Crafting/Graphs/bat.yml

```diff
@@ -0,0 +1,13 @@
+﻿- type: constructionGraph
+  id: bat
```

Comment:
this graph could be given a more verbose ID, other stuff like metal bats exist and might need to be crafted in the future, maybe just `woodenBat` or `WoodenBat`